pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2553,6 +2560,32 @@ func handlePhysicalIOAdapterListImpl(ctxArg interface{}, key string,
 			// We assume AddOrUpdateIoBundle will preserve any
 			// existing IsPort/IsPCIBack/UsedByUUID
 			aa.AddOrUpdateIoBundle(log, *ib)
+
+			// Adding PF before VF to have correct boot order
+			if ib.Type == types.PhyIoNetEthPF && ib.Vfs.Count > 0 {
+				exists, ifName := types.PciLongToIfname(log, ib.PciLong)
+				if !exists {
+					log.Fatal("Failed to resolve ifname for PCI adress ", ib.PciLong)
+				}
+
+				err = createVF(ifName, ib.Vfs.Count)
+				if err != nil {
+					log.Fatal("Failed to create VF for iface with PCI address", ib.PciLong)
+				}
+
+				vfs, err := getVfByTimeout(30*time.Second, ifName)
+				if err != nil {
+					log.Fatal("Failed to get VF for iface ", ifName, " ", err)
```

Comment:
We can leave these log fatals for now since they are only in the SR-IOV path but I'd be slightly more comfortable with a timeout of 150 seconds (less than the watchdog keepalive check at 300 seconds).

Reply:
Done